clients/export/export.go

```diff
@@ -110,3 +110,27 @@ func (c Client) ExportAll(ctx context.Context, params *AllParams) error {
 	}
 	return nil
 }
+
+type StackParams struct {
+	OutParams
+	StackId 	string
+}
+
+func (c Client) ExportStack(ctx context.Context, params *StackParams) error {
+	if params.StackId == "" {
+		return clients.ErrMustSpecifyStack
+	}
+
+	stackId := params.StackId
```

Comment:
Nitpick: this local var isn't really needed, you could use `params.StackId` everywhere.